src/Components/CreateImageWizard/formComponents/ActivationKeyInformation.js

```diff
@@ -60,19 +60,19 @@ const ActivationKeyInformation = () => {
               Role:
             </TextListItem>
             <TextListItem component={TextListItemVariants.dd}>
-              {activationKeyInfo.body?.role || 'Not defined'}
+              {activationKeyInfo.role || 'Not defined'}
```

Comment:
This is actually still going to need to be `activationKeyInfo.body.role`. It seems a bit clunky, but that's how their API works... and right now, I want to move towards just being 1:1 with the APIs that we consume and not doing any sort of transformations or normalizations. Less decision making that way, which hopefully will mean less errors. :sweat_smile: 

I think it would actually be best to just convert ActivationKeyInfo.js into a .tsx file in this PR. That will help you ensure that all the properties that are accessed actually exist. :wink:

Reply:
Hmm, let's split the migration of the activation key to a different PR then.

Reply:
We could just do it as a commit in this PR, too. I checked and the necessary changes are really pretty minor. It's up to you! :grin: